infra/src/uber-charts/islandis.ts

```diff
@@ -168,4 +171,5 @@ export const FeatureDeploymentServices = []
 export const ExcludedFeatureDeploymentServices = [
   userNotificationService,
   userNotificationWorkerService,
+  contentfulEntryTagger,
```

Comment:
Just FYI - this means the service cannot be feature-deployed.

Reply:
That's fine, I don't see a reason for it to be feature deployed